test/api_integration/apis/saved_objects_management/scroll_count.ts

```diff
@@ -18,77 +18,132 @@ export default function ({ getService }: FtrProviderContext) {
-        .post(apiUrl)
-        .send({
-          typesToInclude: defaultTypes,
-        })
-        .expect(200);
-
-      expect(res.body).to.eql({
-        dashboard: 2,
-        'index-pattern': 1,
-        search: 1,
-        visualization: 2,
+        expect(res.body).to.eql({
+          dashboard: 2,
+          'index-pattern': 1,
+          search: 1,
+          visualization: 2,
+        });
       });
-    });
 
-    it('only returns count for types to include', async () => {
-      const res = await supertest
-        .post(apiUrl)
-        .send({
-          typesToInclude: ['dashboard', 'search'],
-        })
-        .expect(200);
-
-      expect(res.body).to.eql({
-        dashboard: 2,
-        search: 1,
+      it('only returns count for types to include', async () => {
+        const res = await supertest
+          .post(apiUrl)
+          .send({
+            typesToInclude: ['dashboard', 'search'],
+          })
+          .expect(200);
+
+        expect(res.body).to.eql({
+          dashboard: 2,
+          search: 1,
+        });
       });
-    });
 
-    it('filters on title when `searchString` is provided', async () => {
-      const res = await supertest
-        .post(apiUrl)
-        .send({
-          typesToInclude: defaultTypes,
-          searchString: 'Amazing',
-        })
-        .expect(200);
-
-      expect(res.body).to.eql({
-        dashboard: 1,
-        visualization: 1,
-        'index-pattern': 0,
-        search: 0,
+      it('filters on title when `searchString` is provided', async () => {
+        const res = await supertest
+          .post(apiUrl)
+          .send({
+            typesToInclude: defaultTypes,
+            searchString: 'Amazing',
+          })
+          .expect(200);
+
+        expect(res.body).to.eql({
+          dashboard: 1,
+          visualization: 1,
+          'index-pattern': 0,
+          search: 0,
+        });
+      });
+
+      it('includes all requested types even when none match the search', async () => {
+        const res = await supertest
+          .post(apiUrl)
+          .send({
+            typesToInclude: ['dashboard', 'search', 'visualization'],
+            searchString: 'nothing-will-match',
+          })
+          .expect(200);
+
+        expect(res.body).to.eql({
+          dashboard: 0,
+          visualization: 0,
+          search: 0,
+        });
       });
     });
 
-    it('includes all requested types even when none match the search', async () => {
-      const res = await supertest
-        .post(apiUrl)
-        .send({
-          typesToInclude: ['dashboard', 'search', 'visualization'],
-          searchString: 'nothing-will-match',
-        })
-        .expect(200);
-
-      expect(res.body).to.eql({
-        dashboard: 0,
-        visualization: 0,
-        search: 0,
+    describe('scroll_count with more than 10k objects', () => {
+      const importVisualizations = async ({
+        startIdx = 1,
+        endIdx,
```

Comment:
Only added FTR tests for this API because `export` and `import` were already properly working and are covered, e.g 

https://github.com/elastic/kibana/blob/7917e3c9d924f35b75f0c213904dd393a9256969/test/api_integration/apis/saved_objects/export.ts#L525-L527